helloworld/Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/groovy
+node('redhatdistortion-maven') {
+    stage 'Checkout'
+    checkout scm
```

Comment:
I get your point. But without that step, it does not work. This might be because of the fact that you can have Jenkinsfile and source from different branches even repositories.